DB/src/main/java/io/deephaven/db/util/PythonScopeJpyImpl.java

```diff
@@ -67,6 +67,72 @@ public CallableWrapper(PyObject pyObject) {
         public Object call(Object... args) {
             return convert(pyObject.callMethod("__call__", args));
         }
+
+        public PyObject getPyObject() {
+            return pyObject;
+        }
+    }
+
+    public static final class NumbaCallableWrapper extends CallableWrapper {
+        private List<Class> paramTypes;
+        private Class returnType;
+
+        public NumbaCallableWrapper(PyObject pyObject, Class returnType, List<Class> paramTypes) {
+            super(pyObject);
+            this.returnType = returnType;
+            this.paramTypes = paramTypes;
+        }
+
+        public Class getReturnType() {
+            return returnType;
+        }
+
+        public List<Class> getParamTypes() {
+            return paramTypes;
+        }
+    }
+
+    private static CallableWrapper wrapCallable(PyObject pyObject) {
+        if (pyObject.hasAttribute("ufunc")) {
```

Comment:
I wonder if we should try to be more type safe here, and check the actual type against `numpy.ufunc`?

Reply:
At the very least, I think we should add a comment or link to the ufunc documentation.

Reply:
in the new commit, python classname is used to to make it more type safe